srcpkgs/llvm7/template

```diff
@@ -13,37 +13,22 @@ configure_args="
  -DLLVM_ENABLE_FFI=ON
  -DLLVM_BINUTILS_INCDIR=/usr/include"
 short_desc="Low Level Virtual Machine"
-maintainer="Juan RP <xtraeme@voidlinux.org>"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
```

Comment:
needs to be revbumped

Reply:
Ah, yes, that went lost during the rebasing, @maxice8